homeworks/roman.plynko_RPVanarx/homework_3/task2.js

```diff
@@ -0,0 +1,43 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+let searchPowNumber;
+let minValue;
+let maxValue;
+
+function searchPow(n, min, max) {
+    if (n >= min && n <= max) {
```

Comment:
1. make you function to take only one argument (number to check), and handle the rest with a cycle, now you pass min and max values  each time, but these values never change

2. also keep all data you need for function execution inside function body, unless they have specific purpose to be in outer scope

this code may work, but it needs decent optimization